Permanent/Modules/EditMetadata/ViewModels/AppendFilenameViewModel.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  AppendFilenameViewModel.swift
+//  Permanent
+//
+//  Created by Lucian Cerbu on 02.08.2023.
+
+import Foundation
+
+class AppendFilenameViewModel: ObservableObject {
+    @Published var textToAppend: String = ""
+    @Published var positionForText: String = ""
+}
+
+enum whereToAppendText: String {
```

Comment:
Enums name should start with an upper case letter.  Also, either move it to it's own file or include it into the ViewModel or in some other struct/class. 

eg:

```
class AppendFilenameViewModel: ObservableObject {
enum whereToAppendText: String {
    case beforeFilename
    case afterFilename

    var position: String {
        switch self {
        case .beforeFilename:
            return "Before filename"
        case .afterFilename:
            return "After filename"
        }
    }
}

    @Published var textToAppend: String = ""
    @Published var positionForText: String = ""
}
```

Reply:
I moved it to the ViewModel.